src/subscript/sector2fluxnum/sector2fluxnum.py

```diff
@@ -0,0 +1,251 @@
+import argparse
+import sys
+import os
+import datetime
+from cwrap import open
+from ecl.grid import EclGrid
+from ecl.eclfile import EclFile, FortIO
+from subscript.sector2fluxnum import flux_obj
+from subscript.sector2fluxnum import fluxfile_obj as ffo
+from subscript.sector2fluxnum import datafile_obj as do
+from subscript.sector2fluxnum import completions as comp
+
+
+DESCRIPTION = """
+This is a wrapper script that executes the different
+steps in the workflow for generating sector models in ECLIPSE.
+
+The sector models have the same resolution as the full-field model
+
+Further details about usage and howto can be found here:
+
+http://wiki.equinor.no/wiki/index.php/Res:Gen_FLUX_fipnum_region.py
+
+Author: Thomas Ramstad, trams@equinor.com .
+
+"""
+
+
+def header_print():
+    """ """
+    sys.stdout.write(
+        "*********************************************************************\n"
+        "*********************************************************************\n"
+        "** This is a wrapper script for making sector models               **\n"
+        "** with full field resulution.                                     **\n"
+        "**                                                                 **\n"
+        "** Never forget to QC results of a script: The user is responsible **\n"
```

Comment:
Remove line 37 and 38. This is steering docs stuff and we should avoid touching that subject in source code..